.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: node_js
+node_js:
+- '8'
+- '9'
+- '10'
+cache:
+  directories:
+  - node_modules
+before_script:
+- npm run build
+- npm run build.browser
+script:
+- npm run test
+- npx embedme README.md
+
```

Comment:
We could add as well "before_deploy" to deploy the docs to github_pages: https://github.com/nemfoundation/nem2-hd-wallets/pull/13/files#diff-0fd4ef892d9d4990033701887c2f9bcc